app/lib/actions/pulp/consumer/sync_capsule.rb

```diff
@@ -0,0 +1,20 @@
+module Actions
+  module Pulp
+    module Consumer
+      class SyncCapsule < ::Actions::Pulp::AbstractAsyncTask
+        input_format do
+          param :capsule_id, Integer
```

Comment:
Its not apparent explicitly, but in Actions::Pulp::Abstract, [the capsule id](https://github.com/Katello/katello/pull/5750/files#diff-154d82fd1b13b035e725be99e3cec146R8)  input lets pulp_resources and pulp_extensions call the pulp server on the capsule.